mobile-broadband-provider-info/serviceproviders.xml

```diff
@@ -1266,13 +1266,14 @@ conceived.
 	<provider>
 		<name>Mobile Vikings</name>
 		<gsm>
-			<network-id mcc="206" mnc="20"/>
+			<network-id mcc="206" mnc="01"/>
 			<voicemail>1933</voicemail>
-			<apn value="web.be">
+			<apn value="internet.proximus.be">
 				<plan type="prepaid"/>
 				<usage type="internet"/>
 				<username>web</username>
 				<password>web</password>
+				<name>Mobile Vikings Internet</name>
```

Comment:
This needs to be below the <usage>. Otherwise the xml doesn't validate and build fails.